### DISCUSSION TITLE
Any way to set retention to the logs in CloudWatch Logs?

### DISCUSSION THREAD

**Discussion by toshitanian** [aws/copilot-cli#3988] [Q&A]
Thanks to Copilot CLI, our service deployed using it is in the production environment!
Now we found that the logs from the services has not retention period, so we have to clean them up by ourself. 
Is there already any way to set retention period while we define the manifest?

<img width="1236" alt="Screen Shot 2022-09-10 at 14 58 03" src="https://user-images.githubusercontent.com/1647045/189471370-a37524a0-6ca6-46f5-908e-20f0d70d4f9c.png">

**Comment by iamhopaul123** [comment]
Hello @toshitanian. We do have a field in the workload manifest for [log retention](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#logging-retention) but that's for the service log group instead of the log of lambdas that we used to better deploy the stack. It would be a good enhancement request for setting retention for them as well, as most of them won't make any sense after 30 or 60 days.

Opened an issue tracking this: https://github.com/aws/copilot-cli/issues/3992. Thank you for the feedback!

**Reply by toshitanian** [reply]
Thank you for making an issue ! Will handle this issue by ourselves for a while!
